scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Api.scala

```diff
@@ -1,6 +1,64 @@
 package scala.meta
 package semantic
 
-private[meta] trait Api extends v1.Api
+import scala.meta.inputs._
 
-private[meta] trait Aliases extends v1.Aliases
+private[meta] trait Api extends Flags {
+  implicit class XtensionPositionAnchor(pos: Position) {
+    def toAnchor: Anchor = pos.input match {
+      case scala.meta.inputs.Input.File(path, _) =>
+        Anchor(path, pos.start.offset, pos.end.offset)
+      case other =>
+        sys.error(s"unsupported input " + other)
+    }
+  }
+
+  implicit class XtensionSemanticEquality(tree1: Tree)(implicit m: Mirror) {
+    def ===(tree2: Tree): Boolean = scala.meta.internal.semantic.equality.equals(tree1, tree2)
```

Comment:
My only problem with these names is indeed verbosity. I think there's a reason why methods like `<:<` and `=:=` are called like that. Less frequently used methods like semantic/structural/syntactic equality we can probably keep in their long form.